alternator/server.hh

```diff
@@ -49,6 +50,7 @@ class server {
     alternator_callbacks_map _callbacks;
 
     semaphore* _memory_limiter;
+    utils::updateable_value<uint32_t> _max_concurrent_requests;
```

Comment:
This is the first time I see this "updatable_value" thing... How does it get updated?

Reply:
By re-reading the config file, which is, in turn, triggered by sending a SIGHUP signal to Scylla (`pkill -HUP scylla`).